### DISCUSSION TITLE
Step 4) ilæg-nye-punkter

### DISCUSSION THREAD

**Discussion by ErikLysdal** [Kortforsyningen/FIRE#321] [Q&A]
Findes der en beskrivelse af, hvordan step 4 skal afvikles?
Det har ikke været muligt at udlede kommandoerne med eksisterende beskrivelse.

**Comment by kbevers** [comment]
Der er i hvert fald nogle hints, det kan godt skrives på en smartere måde. Under step 1) i workshoppen står der i den sidste halvdel lidt om hvordan nyoprettede punkter indtastes. Søg på "Ang. nyetablerede punkter" så kommer du til det sted hvor det starter.

Der fremgår det at projektregnearket har et faneblad til indtastning af nye punkter. Indsæt en linje for hvert nyt punkt. Under "Afmærkning" har man følgende muligheder: "ukendt", "bolt", "lodret bolt", "skruepløk" og "ingen". 

Når regnearket er udfyldt køres kommandoen:

```
fire niv ilæg-nye-punkter PROJEKT Bxxxxx
```

**Reply by kbevers** [reply]
Noteret som en ting der skal løses snarest: https://github.com/Kortforsyningen/FIRE/issues/322

**Reply by ReneTandsbjerg** [reply]
Vedr. "Afmærkning" har man følgende muligheder: "ukendt", "bolt", "lodret bolt", "skruepløk" og "ingen". 
så skal tallet der kendertegner bolttypen skrives og ikke navnet. f.eks Alm bolt skrives 2700.

**Reply by busstoptaktik** [reply]
@ReneTandsbjerg - man *kan* skrive tallet, men de menneskelæsbare muligheder du anfører er at foretrække. De oversættes automatisk til tal efter [denne](https://github.com/Kortforsyningen/FIRE/blob/c78e1788b573cef71c8a6bdd7a96acc69a5d4d79/fire/cli/niv/il%C3%A6g_nye_punkter.py#L145-L151) tabel.

**Reply by ReneTandsbjerg** [reply]
Ok.
